client/components/userAccount.jsx

```diff
@@ -107,6 +115,63 @@ class UserAccount extends Component {
                   : null}
               </div>
             </div>
+            <div className="column is-half">
+              <p className="title is-4">My Previous Orders</p>
+              <div id="prevUserOrderBox" className="adminBox">
+                {
+                  // console.log(inactiveOrders)
+                  inactiveOrders.length
+                    // eslint-disable-next-line max-len
+                    ? inactiveOrders.filter((order) => order.username === loggedInUser.username).map((order) => (
+                      <div className="box" key={order.inactiveId}>
+                        <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+                          <p>
+                            {order.username}
```

Comment:
I don't think we need to show username on this page, since it's the user's own orders

Reply:
I was thinking in terms of seeing a receipt in your inbox that has your name on it, but updated to be "Total" up at the top